.infra/application.properties

```diff
@@ -11,8 +11,11 @@ debezium.source.table.include.list=public.comment,public.comment_upvote,public.c
 debezium.source.column.exclude.list=public.post.tsv,public.post.placeholder,public.post.views,public.post.score,public.post.discussionScore,public.notification."readAt"
 debezium.source.plugin.name=pgoutput
 debezium.source.heartbeat.interval.ms=60000
-debezium.transforms=Reroute
+debezium.transforms=Reroute,filter
 debezium.transforms.Reroute.type=io.debezium.transforms.ByLogicalTableRouter
 debezium.transforms.Reroute.topic.regex=(.*)
 debezium.transforms.Reroute.topic.replacement=%topic%
+debezium.transforms.filter.type=io.debezium.transforms.Filter
+debezium.transforms.filter.language=jsr223.groovy
+debezium.transforms.filter.condition=!(valueSchema.field('op') && value.op == 'u' && value.before && value.before.flags == '{}' && value.after.flags != '{}')
```

Comment:
I'm not familiar with this so can't really comment on whether this is correct or not.
Assume you tested it and it does what it needs to.